Jenkinsfile

```diff
@@ -17,9 +17,7 @@ pipeline {
             when {
                 expression { 
                     GIT_BRANCH == 'master' ||
-                        GIT_BRANCH == 'dev' ||
-                        GIT_BRANCH == 'ci' ||
-                        GIT_BRANCH == 'dev-audit' 
+                        GIT_BRANCH == 'master-pre-merge' ||
```

Comment:
I think the `||` may be causing the ci failure

Reply:
oops, didn't even realize this started failing.  Will fix.